client/cmd/split.go

```diff
@@ -1,31 +1,121 @@
 package cmd
 
 import (
+	"bytes"
 	"context"
 	"encoding/hex"
 	"fmt"
 	"math/big"
 	"os"
 	"strings"
 
+	"github.com/iden3/go-iden3-crypto/poseidon"
 	"github.com/shopspring/decimal"
 	"github.com/spf13/cobra"
 	"source.quilibrium.com/quilibrium/monorepo/node/protobufs"
 )
 
+func getCoinAmount(coinaddr []byte) *big.Int {
```

Comment:
I couldn't find a more efficient way to query the amount of a specific coin address, so I adopted the approach from `client/cmd/coins.go`.

If there's a better way I'm happy to refactor :)